hieradata/clients/controller.sponsorship.ci.jenkins.io.yaml

```
@@ -392,6 +392,44 @@ profile::jenkinscontroller::jcasc:
           credentialsId: "jenkinsvmagents-userpass"
           usePrivateIP: true
           spot: true
+        - name: "ubuntu-22-inbound-amd64-jdk17"
+          description: "Ubuntu 22.04 LTS"
+          imageDefinition: jenkins-agent-ubuntu-22.04-amd64
+          os: "ubuntu"
+          os_version: "22.04"
+          launcher: "inbound"
+          location: "East US 2"
+          instanceType: Standard_D4ads_v5 # 4 vCPUS / 16 Gb / Max 150 Gb local storage
+          ephemeralOSDisk: true
+          architecture: amd64
+          labels:
+            - ubuntu-22-amd64-maven-17
+          javaHome: /opt/jdk-17 # Test override of the default JDK for builds
+          agentJavaBin: /opt/jdk-17 # Test override of the default JDK for builds
+          maxInstances: 50
+          useAsMuchAsPossible: true
+          credentialsId: "jenkinsvmagents-userpass"
+          usePrivateIP: true
+          spot: true
+        - name: "ubuntu-22-amd64-inbound-jdk21"
```

```suggestion
        - name: "ubuntu-22-amd64-maven21"
```

```suggestion
        - name: "ubuntu-22-amd64-maven17"
```

- Remove the `inbound` element as it is an implementation detail (and can be changed in the template). It does not give any useful information to the end user (only to administrators) so better removing it
- Using the same naming as the existing container agent: `maven21` instead of `jdk21` (ref. https://github.com/jenkinsci/jenkins/blob/0d5232fca40a12b2d14f7ded577a90f0cb3c6f06/Jenkinsfile#L22 as example of consumption)